src/Presentation/Nop.Web/Views/Checkout/Confirm.cshtml

```diff
@@ -73,6 +73,14 @@
                     <div class="buttons">
                         <input type="submit" name="nextstep" value="@T("Checkout.ConfirmButton")" class="button-1 confirm-order-next-step-button" />
                     </div>
+					<script>
+                        $(document).on('submit', 'form', function() {
+                            var button = $(this).find('input[type="submit"]');
+                            setTimeout(function() {

```

Comment:
**IsMinimumOrderPlacementIntervalValid** in ConfirmOrder() throws an exception and to display that exception the page reloads. -> The initial state of the button stays enabled.

And on the other side, OpcConfirm is not affected by this.